dispatchers.ts

```diff
@@ -21,6 +22,9 @@ export class CommandDispatcher implements ICommandDispatcher {
 				return;
 			}
 
+			this.$logger.trace("System information:");
+			this.$logger.trace(this.$sysInfo.getSysInfo());
```

Comment:
I guess gathering system information could take time which will slow down each command execution. Can this be `if`ed so that we `getSysInfo` only when tracing.